functions/dailyCC/dailyCC.js

```diff
@@ -1,124 +1,89 @@
-const axios = require("axios");
-const {slugs, baseUrl} = require('./constants')
-const {randomIndex, removeFromList} = require('../utilities')
-
-const getQuestion = async (slug)=>{
-	let url = baseUrl + '/' + slug
-	try{
-		let resp = await axios.get(url)
-		return resp.data
-	}
-	catch(err){
-		throw err
-	}
-}
-
-
-const getUsedSlugs = async (firestore)=>{
-	try{
-		let snapshot = await firestore.collection('dailyCC').doc('slugs').get();
-		if(snapshot.exists){
-			let slugs = snapshot.data()
-			return slugs.usedSlugs
-		}
-		else {
-			throw 'Snapshot does not existt'
-		}
-	}
-	catch(err){
-		throw err
-	}
-
-}
-
-const getPossibleSlugs = async({firestore, ignoreList=[]})=>{
-	try{
-		let snapshot = await firestore.collection('dailyCC').doc('slugs').get();
-
-		if(snapshot.exists){
-			let slugs = snapshot.data()
-			return removeFromList(slugs.allSlugs, ignoreList)
-		}
-		else {
-			throw 'Snapshot does not existt'
-		}
+const axios = require('axios');
+const { firestore, discordMessageEmbed } = require('../config');
+const { slugs, baseUrl, codeWarsLogo } = require('./constants');
+const { randomIndex } = require('../utilities');
+
+const getQuestion = async (slug) => {
+	try {
+		const { data } = await axios.get(`${baseUrl}/${slug}`);
+		return data;
+	} catch (err) {
+		console.log({ getQuestion: err });
 	}
-	catch(err){
-		throw err
+};
+
+const getUsedSlugs = async () => {
+	try {
+		const snapshot = await firestore.collection('dailyCC').doc('slugs').get();
+		return snapshot.data().usedSlugs || [];
```

Comment:
This technically works here.... but isn't necessarily the best way to do it. We should be checking if the snapshot exists or not. If it doesn't, there could be a bigger issue for why it doesn't exist that we might want to check here.

Reply:
The only time the snapshot won't exist is if that doc is manually deleted so why do we need to check for that?

Reply:
In case it is. Yes, as of right now, it never "should" be. But it only benefits us to add a check for this to ensure we would catch it if it ever did.